packages/amplify-migration-tests/src/aws-matchers/iamMatcher.ts

```diff
@@ -0,0 +1,31 @@
+import { IAM } from 'aws-sdk';
```

Comment:
Do we have a similar matcher in e2e-tests

Reply:
Removed